src/Stratis.Bitcoin.IntegrationTests/APITests.cs

```diff
@@ -89,40 +66,35 @@ public void CanGetGeneralInfoViaAPI()
         [Fact]
         public void CanStartStakingViaAPI()
         {
-            try
+            using (NodeBuilder builder = NodeBuilder.Create())
             {
-                var fullNode = this.apiTestsFixture.stratisStakeNode.FullNode;
-                var apiURI = fullNode.NodeService<ApiSettings>().ApiUri;
+                var posNode = builder.CreateStratisPosApiNode();
+                posNode.Start();
```

Comment:
Maybe reduce a line - pass in `start `param as true ? `var posNode = builder.CreateStratisPosApiNode(true);`

Reply:
Fixed 👍